quail/io.py

```diff
@@ -0,0 +1,36 @@
+from pywps import LiteralInput, ComplexInput, ComplexOutput
+from pywps.inout.formats import Format
+
+climdex_input = ComplexInput(
+    "climdex_input",
+    "climdexInput",
+    abstract="Rdata (.rda) file containing R Object of type climdexInput",
+    supported_formats=[
+        Format("application/x-gzip", extension=".rda", encoding="base64")
+    ],
+)
+
+ci_name = LiteralInput(
+    "ci_name",
+    "climdexInput name",
+    abstract="Name of the climdexInput object",
+    min_occurs=1,
+    max_occurs=1,
+    data_type="string",
+)
```

Comment:
I think this input shouldn't be optional. I have it as a mandatory input in my branch, so whoever merges later can fix this.

Reply:
does `min_occurs=1` mean optional? I thought it meant there had to be at least one input

Reply:
Oh, sorry. I misread the code. So that is fine, but I think other inputs don't have that `min_occurs=1` and `max_occurs=1` limitations. I think it would be better if we set those for all of our inputs since we strictly need one for each.